examples/android/app/src/main/jni/main.cpp

```diff
@@ -115,39 +115,41 @@ int main(int argc, char* argv[])
     {
         for (sf::Event event; active ? window.pollEvent(event) : window.waitEvent(event);)
         {
-            switch (event.type)
+            switch (event.getType())
             {
-                case sf::Event::Closed:
+                case sf::Event::Type::Closed:
```

Comment:
I don't see a way around the extra `Type::` here, but I think this is still my least favorite part in this change.